flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -251,4 +252,9 @@ class SentryFlutterOptions extends SentryOptions {
       useFlutterBreadcrumbTracking();
     }
   }
+
+  /// Setting this to a custom [BindingWrapper] allows you to use a custom [WidgetsBinding].
+  @experimental
```

Comment:
If its `internal`, we dont need to use `experimental`

Reply:
It should be experimental, but not internal. After all it's designed to enable the use of custom WidgetsBindings.